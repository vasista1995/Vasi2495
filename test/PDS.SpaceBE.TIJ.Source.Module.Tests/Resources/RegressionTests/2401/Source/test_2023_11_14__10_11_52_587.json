{
  "Module": "SP",
  "Segment": "Tijuana",
  "ProcessEquipment": "WEOA0102",
  "ParameterClass": "E_SPC",
  "Category": "PBF",
  "Classification": "SPC-Monitoreo",
  "Operator": "cerna",
  "Layout": "8IN_ELLEMEX",
  "PartNumber": "913520DH",
  "Area": "Pre_assy_Pasivacion",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096853,
      "Seqnr": 3,
      "Value": 0.041,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096853,
      "Seqnr": 1,
      "Value": 0.045,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096853,
      "Seqnr": 2,
      "Value": 0.04,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "UniqueSourceKey": "TIJ:BE:104OOFV.1:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceXa:1563:3096853:0",
  "PKey": "TIJ:BE:104OOFV.1:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceXa:1563:3096853:0",
  "ParameterName": "PasivationAperture_DistanceXa",
  "ParameterUnit": "mm",
  "SampleDate": "2023-10-09T19:52:33Z",
  "SampleDateUtc": "2023-10-10T02:52:33Z",
  "SampleSize": 3,
  "Sigma": ",00264575131106459",
  "Mean": ",042",
  "Median": ",041",
  "Min": ",04",
  "Max": ",045",
  "Q1": ",0405",
  "Q3": ",043",
  "LdsID": "2401",
  "IsCalcParam": 0,
  "MeanCntrlLow": 0.0297,
  "MeanCntrlTarget": 0.03896,
  "MeanCntrlHigh": 0.04822,
  "RangeCntrlLow": 0.0,
  "RangeCntrlTarget": 0.005025,
  "RangeCntrlHigh": 0.01005,
  "SpecLow": 0.03,
  "SpecTarget": 0.045,
  "SpecHigh": 0.06,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "N",
  "CtrlSigmaLowEnabled": "Y",
  "CtrlSigmaCenterEnabled": "Y",
  "CtrlRangeCenterEnabled": "N",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "Y",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "Y",
  "CreatedDate": "2023-10-09T19:52:34Z",
  "CreatedDateUtc": "2023-10-10T02:52:34Z",
  "UpdatedDate": "2023-10-09T19:52:34Z",
  "UpdatedDateUtc": "2023-10-10T02:52:34Z",
  "RvStoreFlag": "Y",
  "MeasurementEquipment": "MSNI0001",
  "Lot": "104OOFV.1",
  "ChannelId": "1563",
  "CkcId": "0",
  "ChannelName": "PasivationAperture_DistanceXa/WDSON/PBF/8IN_ELLEMEX",
  "SpaceInstanceName": "BE",
  "Sitekey": "TIJ",
  "CalcCreated": "10/05/2018 09:37:08",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 0.0420952380952381,
  "ExtEwmaMV": 0.0420802083029409,
  "ExtMsMV": 0.000235702260395514,
  "OutOfOrder": "N",
  "EwmaS": 0.00229072877004787,
  "EwmaR": 0.00428641656841036,
  "Calc2ID": 2289,
  "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN",
  "ExtEwmaMVLCL": 0.0,
  "ExtEwmaMVCenter": 0.0,
  "ExtEwmaMVUCL": 0.0,
  "ExtMaMVLCL": 0.0,
  "ExtMaMVCenter": 0.0,
  "ExtMaMVUCL": 0.0,
  "EwmaSLCL": 0.0,
  "EwmaSCenter": 0.0,
  "EwmaSUCL": 0.0,
  "EwmaRLCL": 0.0,
  "EwmaRCenter": 0.0,
  "EwmaRUCL": 0.0,
  "ExtMSMVLCL": 0.0,
  "ExtMSMVCenter": 0.0,
  "ExtMSMVUCL": 0.0,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 0.0,
  "RawUal": 0.0,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChState": "Online"
}