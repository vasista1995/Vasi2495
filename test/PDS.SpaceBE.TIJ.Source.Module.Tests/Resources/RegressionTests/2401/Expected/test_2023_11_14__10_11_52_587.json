{
  "IdSource": "SPACEACT2:TIJ:BE:104OOFV.1:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceXa:1563:3096853:0",
  "Data_flat_Metadata": {
    "Segment": "Tijuana",
    "Module": "SP",
    "ParameterClass": "E_SPC",
    "ProcessEquipment": "WEOA0102",
    "Category": "PBF",
    "Classification": "SPC-Monitoreo",
    "Operator": "cerna",
    "Layout": "8IN_ELLEMEX",
    "PartNumber": "913520DH",
    "Area": "Pre_assy_Pasivacion",
    "Lot": "104OOFV.1",
    "ParameterName": "PasivationAperture_DistanceXa",
    "ParameterUnit": "mm",
    "SiteKey": "TIJ",
    "ChannelId": "1563",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-10T02:52:34Z",
    "CreatedTimestamp": "2023-10-09T19:52:34Z",
    "UpdatedTimestamp": "2023-10-09T19:52:34Z",
    "UpdatedTimestampUtc": "2023-10-10T02:52:34Z",
    "ExportedTimestampUtc": "2023-10-09T19:52:33Z",
    "ExportedTimestamp": "2023-10-09T19:52:33Z",
    "SampleTimestampUtc": "2023-10-10T02:52:33Z",
    "SampleTimestamp": "2023-10-09T19:52:33Z",
    "SourceDataLevel": "C",
    "Equipment": "MSNI0001",
    "ChannelName": "PasivationAperture_DistanceXa/WDSON/PBF/8IN_ELLEMEX",
    "RvStoreFlag": "Y",
    "CalcCreated": "10/05/2018 09:37:08",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2289,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2401",
    "IsCalcParam": 0,
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 0.06,
        "SpecLow": 0.03,
        "SpecTarget": 0.045,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": ",042",
        "Median": ",041",
        "Sigma": ",00264575131106459",
        "Min": ",04",
        "Max": ",045",
        "Q1": ",0405",
        "Q3": ",043"
      },
      "ControlLimits": {
        "ctrl_mean_low": 0.0297,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 0.03896,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 0.04822,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "Y",
        "CtrlSigmaHighEnabled": "Y",
        "ctrl_range_low": 0.0,
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.005025,
        "ctrl_range_high": 0.01005,
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 0.0420952380952381,
        "ExtEwmaMV": 0.0420802083029409,
        "ExtMsMV": 0.000235702260395514,
        "EwmaS": 0.00229072877004787,
        "EwmaR": 0.00428641656841036,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 3096853,
        "Seqnr": 3,
        "Value": 0.041,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096853,
        "Seqnr": 1,
        "Value": 0.045,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096853,
        "Seqnr": 2,
        "Value": 0.04,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 3
  },
  "ProductionAction": {
    "Id": "SPACEACT1:TIJ:BE:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceXa:1563",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:104OOFV.1",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "TIJ:BE:104OOFV.1:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceXa:1563:3096853:0"
  }
}